scheduler/scheduler-server/src/main/java/org/ow2/proactive/scheduler/core/SchedulingService.java

```diff
@@ -521,6 +537,36 @@ public Boolean call() throws Exception {
         }
     }
 
+    public boolean restartInErrorTask(final JobId jobId, final String taskName)
+            throws UnknownJobException, UnknownTaskException {
+        try {
+            if (status.isUnusable()) {
+                return false;
+            }
+
+            Boolean result = infrastructure.getClientOperationsThreadPool().submit(new Callable<Boolean>() {
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Immediately return this expression instead of assigning it to the temporary variable "result". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1488)